net/nginx/Makefile

```diff
@@ -523,18 +523,19 @@ endif
 
 ifeq ($(CONFIG_NGINX_DAV),y)
   define Download/nginx-dav-ext-module
-    VERSION:=430fd774fe838a04f1a5defbf1dd571d42300cf9
+    VERSION:=f5e30888a256136d9c550bf1ada77d6ea78a48af
     SUBDIR:=nginx-dav-ext-module
     FILE:=nginx-dav-ext-module-$$(VERSION).tar.gz
```

Comment:
xz can be used to save space

Reply:
How to download xz from github?

Reply:
Just change gz to xz, the archive is created locally. See commit openwrt/packages@7d052e50123b5f54bfeb9e968b20e30e4a767730